Formula/ccm.rb

```diff
@@ -19,13 +19,18 @@ class Ccm < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "ae80a7c1a071309c01de99d929e68b38c969efc884dbbac25da2ff6fb6e8b162"
   end
 
-  depends_on "python@3.10"
+  depends_on "python@3.11"
   depends_on "pyyaml"
   depends_on "six"
 
-  resource "cassandra-driver" do
-    url "https://files.pythonhosted.org/packages/19/bd/b522b200e8a7cc5ace859e9667308a3a302a23d6df09ae087ca2dfbf60c2/cassandra-driver-3.22.0.tar.gz"
-    sha256 "df825ee4ebb7f7fa33ab028d673530184fe0ee41ea66b2f9ddd478db56145a31"
```

Comment:
Also, we may want to figure out why `cassandra-driver` was added -- #32635

Reply:
Seems like this was to support `ccm <node> cqlsh`, which executes a `cqlsh` downloaded by `ccm` and does require `cassandra-driver`: https://github.com/apache/cassandra/blob/d0e0e20f80f7d5a4b37d542b65f2a781ce1d3015/bin/cqlsh.py#L66-L73